stream-chat-android-ui-components/build.gradle

```diff
@@ -0,0 +1,127 @@
+import com.getstream.sdk.chat.Dependencies
+import com.getstream.sdk.chat.Configuration
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'com.hiya.jacoco-android'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'de.mannodermaus.android-junit5'
+apply plugin: 'org.jlleitschuh.gradle.ktlint'
+apply plugin: 'com.github.dcendents.android-maven'
+
+group = Configuration.artifactGroup
+version = Configuration.versionName
+
+android {
+    compileSdkVersion Configuration.compileSdkVersion
+
+    defaultConfig {
+        versionName Configuration.versionName
+
+        buildConfigField 'String', 'STREAM_CHAT_UI_COMPONENTS_VERSION', "\"$version\""
+
+        minSdkVersion Configuration.minSdkVersion
+        targetSdkVersion Configuration.targetSdkVersion
+        vectorDrawables.useSupportLibrary = true
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+            unitTests.returnDefaultValues = true
+            // Show the result of every unit test, even if it passes.
+            all {
+                testLogging {
+                    events 'passed', 'skipped', 'failed', 'standardOut', 'standardError'
+                }
+            }
+        }
+    }
+
+    sourceSets {
+        all {
+            it.java.srcDir "src/$it.name/kotlin"
+        }
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+        noStdlib = true
+    }
+}
+
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions {
+        freeCompilerArgs += [
+                '-progressive',
+                '-Xexplicit-api=strict',
+        ]
+    }
+}
+
+dependencies {
```

Comment:
Until we're not using these dependencies, it would be best not to add them to this module. We'll never review them later, now is the best time to start with a minimal set, and only add what we need as we go.